user-interface/src/main/java/life/qbic/datamanager/views/general/contact/Contact.java

```diff
@@ -22,8 +20,8 @@ public final class Contact implements Serializable {
   private String email;
 
   public Contact(String fullName, String email) {
-    requireNonNull(fullName, "fullName must not be null");
-    requireNonNull(email, "email must not be null");
+//    requireNonNull(fullName, "fullName must not be null");
+//    requireNonNull(email, "email must not be null");
```

Comment:
Why was this commented out? I'd argue a contact should at least provide this information 🤔

Reply:
For this simple object, I do not think this must always be the case. This is only the bean used in the fields in the ui. Require non null would mean that no field bound to this class can be cleared.